### ISSUE TITLE
Emerald, Lapis, Redstone, and Iron Fountain Gadgets.

### ISSUE THREAD

**Issue by TheIntelloBox** [OnlyNoobDied/GadgetsMenu#133]
Hello ! :D 

Could you add this gadgets or could you add a new category of gadgets, with this items ? :D 

Thanks ! :D

**Comment by Waffles2112** [comment]
what if u could combine them into one gadget and call it the "ore fountain" gadget? as all those individual gadgets seems a little excessive

**Comment by TheIntelloBox** [comment]
Yes, you're right, I love the idea ! I have another idea : when you do a right-click on the gadget : you can choose which ore you want to have in the Fountain, and you do a left-click the gadget is enabled.

**Comment by TheIntelloBox** [comment]
Added in the next update ?
